modules/ossm-threescale-manifests.adoc

```diff
@@ -23,7 +23,7 @@ oc exec -n ${NS} $(oc get po -n ${NS} -o jsonpath='{.items[?(@.metadata.labels.a
 .. Hard code the handler as `service_id`
 
 
-. xref:../threescale_adapter/threescale-adapter.html#ossm-threescale-routing_threescale-adapter[Update the workload] with the required annotations:
+. xref:../threescale_adapter/threescale-adapter.adoc#ossm-threescale-routing_threescale-adapter[Update the workload] with the required annotations:
```

Comment:
We haven't been using `xref` in modules:

https://redhat-documentation.github.io/modular-docs/#modular-docs-terms-definitions

Reply:
@jboxman I reverted this change, as this part of the documentation is handled by the 3scale team. @dfennessy please make a note of this for your documentation.